pkg/services/temp_user/model.go

```diff
@@ -24,7 +24,7 @@ const (
 
 // TempUser holds data for org invites and unconfirmed sign ups
 type TempUser struct {
-	ID              int64 `xorm:"id"`
+	ID              int64
```

Comment:
I think this will become `i_d` if it's ever used, I would guess it should be something like this, no?

```suggestion
	ID              int64 `xorm:"pk autoincr 'id'"`
```

Reply:
Sofia got there before me 🎉 🙈